docs/api/loaders/TextureLoader.html

```diff
@@ -54,6 +54,8 @@ <h2>Example with Callbacks</h2>
 		);
 		</code>
 
+		Please note that XHR progress events are not triggered as of Three.js version 88. If you need XHR progress events, you may wish to use Three.js version 83 or [link:https://github.com/mrdoob/three.js/issues/10439#issuecomment-275785639 use one of these loaders].
```

Comment:
>  as of Three.js version 88

I would remove that. Otherwise we'll have to update it every release 😅

Reply:
Indeed. I also think that the wording is not optimal. The impression could arise that only `progress` events are not supported. But `TextureLoader` does not use `XMLHttpRequest` at all. It utilizes `ImageLoader` which is based on the HTML <img> element an its JavaScript representative.

Reply:
@mrdoob @Mugen87 Thanks for this discussion, and I agree on both fronts (though it's confusing the event callbacks for the sample code take` xhr` as their args). How about:

> Please note Three.js version r84 dropped support for `TextureLoader` progress events. If you are using a more modern version of Three.js, you may wish to use one of [these texture loaders](https://github.com/mrdoob/three.js/issues/10439#issuecomment-275785639).

Reply:
The new text is better than before. And it's good that you mentioned the code section. Maybe we should change it to something like this?
```javascript
loader.load(
	// resource URL
	'textures/land_ocean_ice_cloud_2048.jpg',
	// onLoad callback
	function ( texture ) {
		var material = new THREE.MeshBasicMaterial( {
			map: texture
		 } );
	},
	// onProgress callback currently not supported
	undefined,
	// onError callback
	function ( event ) {
		console.error( 'An error happened:', event );
	}
);
```
We should also change the documentation for `ImageLoader`.

Reply:
@Mugen87 Amen! I updated `ImageLoader` too.

While I was in there I updated the docs that extend the `FileLoader` by indicating the default argument to the `onError` callback is the error object caught by the `FileLoader`.

I also standardized the whitespace of the loader examples (both indenting and line breaks were inconsistent) to make them a little easier to read.

Reply:
Great!